<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="suggestionKillsTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>521</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaadot</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="suggestionTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>250</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="suggestionPortionComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="suggestionSuggestPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ehdota</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suggestionCancelPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suggestionSavePushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tallenna</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ruhon osa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ehdotus</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="suggestionDateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Jakopäivä</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>suggestionKillsTableWidget</tabstop>
  <tabstop>suggestionDateEdit</tabstop>
  <tabstop>suggestionPortionComboBox</tabstop>
  <tabstop>suggestionSuggestPushButton</tabstop>
  <tabstop>suggestionTextEdit</tabstop>
  <tabstop>suggestionCancelPushButton</tabstop>
  <tabstop>suggestionSavePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
